deps/src/test/java/tests/unit/com/microsoft/azure/sdk/iot/deps/serializer/FileUploadCompletionNotificationTest.java

```diff
@@ -77,39 +77,18 @@ private static String createJson(String correlationId, Boolean isSuccess, Intege
     public void constructor_succeed()
     {
         // act
-        FileUploadStatusParser fileUploadStatusParser = new FileUploadStatusParser(VALID_CORRELATION_ID, VALID_IS_SUCCESS, VALID_STATUS_CODE, VALID_STATUS_DESCRIPTION);
+        FileUploadCompletionNotification fileUploadStatusParser = new FileUploadCompletionNotification(VALID_CORRELATION_ID, VALID_IS_SUCCESS, VALID_STATUS_CODE, VALID_STATUS_DESCRIPTION);
 
         // assert
         assertFileUploadStatus(fileUploadStatusParser, VALID_CORRELATION_ID, VALID_IS_SUCCESS, VALID_STATUS_CODE, VALID_STATUS_DESCRIPTION);
     }
 
-    /* Tests_SRS_FILE_UPLOAD_STATUS_21_003: [If one of the provided parameters is null, empty, or not valid, the constructor shall throws IllegalArgumentException.] */
-    @Test
-    public void constructor_failed()
-    {
-        for (TestParameters test:tests)
-        {
-            // act
-            try
-            {
-                new FileUploadStatusParser(test.correlationId, test.isSuccess, test.statusCode, test.statusDescription);
-                System.out.println("Test failed: correlationId=" + test.correlationId + ", isSuccess=" + test.isSuccess +
-                        ", statusCode=" + test.statusCode + ", statusDescription=" + test.statusDescription);
-                assert false;
-            }
-            catch (IllegalArgumentException expected)
-            {
-                // Don't do anything, expected throw.
-            }
-        }
-    }
-
     /* Tests_SRS_FILE_UPLOAD_STATUS_21_004: [The toJson shall return a string with a json that represents the contend of the FileUploadStatusParser.] */
```

Comment:
`FileUploadStatusParser` -> `FileUploadCompletionNotification`

Reply:
Actually, I should just delete these comments. They are a relic of an out-dated practice we used to do. They aren't worth keeping around anymore